packages/lib-utils/package.json

```
@@ -37,21 +29,6 @@
     "typesafe-actions": "^4.4.2"
   },
   "devDependencies": {
-    "@monorepo/common": "0.0.0-fixed",
-    "@openshift/dynamic-plugin-sdk": "link:../lib-core",
-    "@rollup/plugin-node-resolve": "^13.1.1",
-    "@rollup/plugin-typescript": "^8.3.0",
-    "@types/lodash-es": "^4.17.5",
-    "@types/pluralize": "^0.0.29",
-    "@types/react": "^17.0.37",
-    "react": "^17.0.2",
-    "react-redux": "^7.2.2",
-    "redux": "^4.1.2",
-    "redux-thunk": "^2.4.1",
-    "rollup": "^2.61.1",
-    "rollup-plugin-analyzer": "^4.0.0",
-    "rollup-plugin-dts": "^4.0.1",
-    "tslib": "^2.3.1",
-    "typescript": "~4.4.4"
+    "@openshift/dynamic-plugin-sdk": "link:../lib-core/dist"
```

We can now use `*` in here. No need to use the `link` value.
```suggestion
    "@openshift/dynamic-plugin-sdk": "*"
```

---

@karelhala This dev-dependency results in Yarn creating `packages/lib-utils/node_modules/@openshift/dynamic-plugin-sdk` specifically pointing to the generated distributable output of `lib-core` package.

I believe this is what we want in general - when building `lib-utils` which has a peer-dependency on `lib-core`, consume `lib-core` via its generated distributable outputs (same behavior as when used by specific plugins or host apps).

---

I am more of a concerned how that looks when published to NPM. Wouldn't this cause an issue with unknown version? But since this is just dev dependency I don't think we have to spend more time on this. The code will be bundled in the final build after all.